src/components/AmountFilter.jsx

```diff
@@ -10,7 +10,7 @@ export default function AmountFilter(props) {
   const error = max > 0 && max < min;
 
   return (
-    <Grid container justifyContent="space-between">
+    <Grid container spacing={2}>
```

Comment:
change necessary for mobile sreens when the filter panel takes the entire width of the screen. Before the `Textbox`s were too spaced out and would span entire width.

Reply:
<img width="878" alt="Screen Shot 2022-02-02 at 6 07 17 PM" src="https://user-images.githubusercontent.com/40483569/152269500-04693036-758c-4a2e-a978-adf31b1fcb1a.png">

Reply:
to prevent the text fields from being spaced out too much